src/Hooks/Dispatchers/CommandEventHookDispatcher.php

```diff
@@ -27,7 +27,7 @@ public function callCommandEventHooks(ConsoleCommandEvent $event)
         $commandEventHooks = $this->getHooks($hooks);
         foreach ($commandEventHooks as $commandEvent) {
             if ($commandEvent instanceof EventDispatcherInterface) {
-                $commandEvent->dispatch(ConsoleEvents::COMMAND, $event);
+                $commandEvent->dispatch($event, ConsoleEvents::COMMAND);
```

Comment:
Does the LegacyEventDispatcherProxy really reverse the order of the parameters to the `dispatch()` method? That seems undesirable for a legacy adapter object.